server/src/Users/Users.controller.ts

```diff
@@ -46,3 +48,29 @@ export const getUsersByOrganization = async (req: Request, res: Response) => {
     res.status(400).json(err.message);
   }
 };
+
+export const deleteUserById = async (req: Request, res: Response) => {
+  try {
+    if (!req.params.id) throw new Error('param is not valid');
+    const users = getRepository(Users);
+    await users.delete({ id: +req.params.id });
+    res.end();
+  } catch (e) {
+    const err = e as Error;
+    res.status(400).json({ message: err.message });
+  }
+};
+
+export const updateUserAdminById = async (req: Request, res: Response) => {
+  try {
+    if (!req.params.id) throw new Error('param is not valid');
+    if (typeof req.body.admin === 'undefined') throw new Error('body is invalid');
+    const users = getRepository(Users);
+    const newAdmin = req.body.admin === 'true' ? true : false;
```

Comment:
그냥 비교연산 결과를 그대로 대입해주셔도 될것같습니다!

```suggestion
    const newAdmin = req.body.admin === 'true';
```